### PR TITLE
Added SYSTEM option to cmake include_dirs

### PR DESCRIPTION
**Pull request by YoshuaNava**
This PR adds the SYSTEM option to cmake include_dirs, which prevents overflow of warnings from external packages included in pointmatcher_ros.

### CODE REVIEW

**Comment by remod** [comment]
To pre-release?

**Comment by YoshuaNava** [comment]
K

File: pointmatcher_ros/src/StampedPointCloud.cpp [hunk 1]
```diff
@@ -171,16 +171,15 @@ bool StampedPointCloud::filterByDistance(const float distanceThreshold, const bo
...
                                             const float zMax, const bool keepInside) {
-  PmPointCloudFilter* boundingBoxFilter = Pm::get().DataPointsFilterRegistrar.create(
-      "BoundingBoxDataPointsFilter", {{"xMin", PointMatcherSupport::toParam(xMin)},
-                                      {"xMax", PointMatcherSupport::toParam(xMax)},
-                                      {"yMin", PointMatcherSupport::toParam(yMin)},
-                                      {"yMax", PointMatcherSupport::toParam(yMax)},
-                                      {"zMin", PointMatcherSupport::toParam(zMin)},
-                                      {"zMax", PointMatcherSupport::toParam(zMax)},
-                                      {"removeInside", PointMatcherSupport::toParam(!keepInside)}});
+  auto boundingBoxFilter = Pm::get().DataPointsFilterRegistrar.create("BoundingBoxDataPointsFilter",
```
Comment (goodfaiter) [inline pointmatcher_ros/src/StampedPointCloud.cpp]: Is it wise to add auto when it is not clear what kind of type is being returned?

Comment (YoshuaNava) [reply pointmatcher_ros/src/StampedPointCloud.cpp]: Old libpointmatcher used to return a raw pointer, and after syncing upstream it returns a shared pointer. This is here just for compatibility.

**Comment by YoshuaNava** [comment]
Ping

**Pull request merged**